cartographer/sensor/data.h

```diff
@@ -41,6 +41,10 @@ struct Data {
     PointCloud ranges;
   };
 
+  struct AbsolutePose {
```

Comment:
The struct is used to distinguish from the odometer_pose, because both of them are Rigid3d, the Data constructor will have the same parameter. We can discuss it if there's other solution.